src/clic-config.adb

```diff
@@ -100,18 +100,27 @@ package body CLIC.Config is
    -- Get_As_String --
    -------------------
 
-   function Get_As_String (This : Instance;
-                           Key : Config_Key)
-                           return String
+   function Get_As_String (This    : Instance;
+                           Key     : Config_Key;
+                           Default : String) return String
```

Comment:
Actually do we even need two function if the `Default` has a default value: `Default : String := ""` ? :)

Reply:
Haha right, indeed :)